tmt/steps/discover/__init__.py

```diff
@@ -192,7 +231,33 @@ def base_command(cls, method_class=None, usage=None):
         @click.option(
             '-h', '--how', metavar='METHOD',
             help='Use specified method to discover tests.')
+        @click.option(
+            '--fmf-id', default=False, is_flag=True,
+            help='Show fmf identifiers for tests discovered in plan.')
         def discover(context, **kwargs):
+            if kwargs.get('fmf_id'):
```

Comment:
Shouldn't whole fmf-id functionality be moved to `DiscoverFmf` class?

Reply:
Thanks for the review Lukas.
I've moved the option `--fmf-id` to `DiscoverFmf` and it helped avoiding the checking of `-h shell --fmf-id`. The `click.Context` module checks it.
Unfortunately it's not possible to move the condition above to `DiscoverFmf` because disabling logging (debug, verbose) should be done immediately after start. In `DiscoverFmf.go()` it's "too late" because some data will be already printed.